### PR TITLE
Feat/update elixir deps

### PR DESCRIPTION
**Pull request by sleipnir**
**Summary of changes**
Changes introduced in this pull request:
- 
Update elixir dependencies

### CODE REVIEW

File: elixir_grpc_bench/config/dev.exs [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import Config
 
-config :falco, start_server: true
+config :grpc, start_server: true
 
 config :logger,
-  level: :error,
-  compile_time_purge_level: :error,
+  level: :debug,
```
Comment (LesnyRumcajs) [inline elixir_grpc_bench/config/dev.exs:6]: won't setting logger to debug hamper the performance? or it doesn't matter here?

Comment (sleipnir) [reply elixir_grpc_bench/config/dev.exs:6]: Hi @LesnyRumcajs. We do not issue messages on the service so it has no effect.

Comment (LesnyRumcajs) [reply elixir_grpc_bench/config/dev.exs:6]: alright, sounds good!

**Review by LesnyRumcajs — APPROVED** [review]
thanks!

**Comment by sleipnir** [comment]
> thanks!

Thanks @LesnyRumcajs

**Pull request merged**